test/hotspot/gtest/logging/test_logOutputList.cpp

```diff
@@ -66,7 +66,6 @@ TEST(LogOutputList, set_output_level_update) {
   size_t outputs_on_level[LogLevel::Count];
   LogLevelType expected_level_for_output[TestOutputCount];
 
-  os::init_random(0x4711);
```

Comment:
I assume this was done to try and get some measure of reproducibility, and now we have lost that, even though it wasn't guaranteed anyway.

Reply:
Yes, I don't know why this was done here.